utils/DaoFactory.js

```diff
@@ -372,7 +371,7 @@ async function cloneDaoDeployer(deployer) {
   const dao = await DaoRegistry.deployed();
   await deployer.deploy(DaoFactory, dao.address);
   let daoFactory = await DaoFactory.deployed();
-  await daoFactory.createDao("test-dao");
+  await daoFactory.createDao("test-dao", ETH_TOKEN);
```

Comment:
Do we want to use the ETH_TOKEN as the factory here?

Reply:
So ... it's just that this address is not important and ETH_TOKEN is 0x0 so this is the only way I could think of to do this